cmd/cluster-agent/api/v1/kubernetes_metadata.go

```diff
@@ -67,7 +68,8 @@ func getNodeMetadata(w http.ResponseWriter, r *http.Request, wmeta workloadmeta.
 	var dataBytes []byte
 	nodeName := vars["nodeName"]
 
-	nodeMetadata, err := wmeta.GetKubernetesMetadata(fmt.Sprintf("nodes//%s", nodeName))
```

Comment:
There are other usages that we should replace:
- https://github.com/DataDog/datadog-agent/blob/5a9eb559df139c233e8bed517c1aab511c3cde21/comp/core/workloadmeta/impl/store_test.go#L1385
- https://github.com/DataDog/datadog-agent/blob/5a9eb559df139c233e8bed517c1aab511c3cde21/pkg/clusteragent/admission/mutate/cwsinstrumentation/cws_instrumentation.go#L360
- https://github.com/DataDog/datadog-agent/blob/5a9eb559df139c233e8bed517c1aab511c3cde21/comp/otelcol/otlp/components/processor/infraattributesprocessor/metrics.go#L67
- https://github.com/DataDog/datadog-agent/blob/5a9eb559df139c233e8bed517c1aab511c3cde21/comp/otelcol/otlp/components/processor/infraattributesprocessor/metrics_test.go#L253


And also for namespaces. Search for "namespaces//" in the code.